docs/balrog/update_to_last_good_build.md

```diff
@@ -0,0 +1,14 @@
+Sometimes bad builds have been shipped and releaseduty must stop users from updating to these builds. Here's the suggested process:
+
+1. Click on the big red button at the top right corner of the page:
+![scheduled change without signoffs](/docs/balrog/big_red_button.png?raw=true)
+This will turn off all updates on this channel. This means any user with any build will be told there is no new update.
+1. Figure out what is the ID of the last good build.
+1. Schedule a rule change to that build
+1. Turn updates back on. From now on, users will update to the most recent good build.
```

Comment:
There should be some indication that this step should happen after the scheduled rule has taken affect or, be scheduled after the above rule has taken affect. (I haven't pushed the button, so I'm not sure exactly how it gets disabled).

Reply:
Nice catch. Added.